docs/tex/HACCP.tex

```diff
@@ -5,28 +5,55 @@ \section{Hazard Analysis and Critical Control Point (HACCP) Plan}
 \subsection{Food Production System Description}
 % Incl. flowchart
 
+
 \subsection{Critical Points}
```

Comment:
New definitional understanding: a critical point is a minimally-complex system (i.e. cannot be realistically/usefully broken down into smaller systems), i.e. a single component that accomplishes a single function that would otherwise be FAILING.

Reply:
Start at the macro-system level (aeroponics, lighting, etc.) and break down from there. List the ones you know AND are included in the solution overview

Reply:
I think what you have here is still good and stuff like it should be included, just add the above to it